screen/accounts/src/main/java/com/ivy/accounts/AccountsTab.kt

```diff
@@ -110,20 +110,12 @@ private fun BoxWithConstraintsScope.UI(
 
                     Spacer(Modifier.height(4.dp))
 
-                    Text(
-                        text = state.totalBalanceWithExcludedText,
-                        style = UI.typo.nB2.style(
-                            color = Gray,
-                            fontWeight = FontWeight.Bold
-                        )
-                    )
-
-                    Text(
-                        text = state.totalBalanceWithoutExcludedText,
-                        style = UI.typo.nC.style(
-                            color = Gray,
-                            fontWeight = FontWeight.Bold
-                        )
+                    IncomeExpensesRow(
+                        currency = state.baseCurrency,
+                        incomeLabel = "Total Balance",
```

Comment:
This string should be extracted as a string resource so the app can be translated